Resources/Prototypes/_Andromeda/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -0,0 +1,26 @@
+#Spartan Hardsuit - adição por SrKolobanov.
+- type: entity
+  parent: ClothingHeadHardsuitWithLightBase
+  id: ClothingHeadHelmetHardsuitSpartan
+  name: spartan hardsuit helmet
+  description: Advanced armored hardsuit helmet for supersoldiers. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)